http/src/main/resources/init-resources/gce-init.sh

```diff
@@ -470,6 +473,11 @@ if [ ! -z "$JUPYTER_DOCKER_IMAGE" ] ; then
   STEP_TIMINGS+=($(date +%s))
 fi
 
+# RStudio specific setup; only do if RStudio is installed
+if [ ! -z "$RSTUDIO_DOCKER_IMAGE" ] ; then
+  retry 3 docker exec ${RSTUDIO_SERVER_NAME} ${RSTUDIO_SCRIPTS}/set_up_package_dir.sh
```

Comment:
This might not be backwards compatible -- e.g. this script won't exist on older RStudio images which would cause the init script to fail. Could we check if the script exists in the image somehow? Alternatively we've used this pattern for things that might fail:
```
EXIT_CODE=0
docker_cmd_that_might_fail || EXIT_CODE=$?
if [ $EXIT_CODE -ne 0 ]; then
  echo "something failed"
fi
```